### DISCUSSION TITLE
How to send Email using SMTP in Jamku?

### DISCUSSION THREAD

**Discussion by yogeshkudikala** [madrecha/jamku#914] [QA] [answered]
Instead of default `noreply-OFFICEID@jamku.app` I want to send email using own email server i.e. SMTP Config.

**Comment by adarshmadrecha** [comment, marked as answer]
## Jamku SMTP Feature.
-  Instead of default `noreply-OFFICEID@jamku.app` now you can send emails to your clients/team members from your own SMTP Config.
- To add your own SMTP config visit `Office Config -> Email -> SMTP config tab`. Fill all the fields and click on update.
- If the SMTP config is not set all the mails will be sent using this default email id `noreply-OFFICEID@jamku.app`.
- While sending emails in the `Send Email` dialog box you'll see an toggle button beside the `Close` button. If `Jamku` is selected then the email will be sent through `noreply-OFFICEID@jamku.app` and if we choose `SMTP` as below then the emails will be sent through your configured SMTP email id.
![image](https://user-images.githubusercontent.com/51023000/150626616-2d7c98bd-50c3-437c-91ca-fb4864e9db03.png)
![image](https://user-images.githubusercontent.com/51023000/150626640-2e2450af-2371-4244-bf14-aa375d38b0c1.png)


## How to configure your own `GMAIL SMTP`

- Host address -> `smtp.gmail.com`
- Port -> `587`
- Email id -> "youremailid"
- Email Password -> "Follow the steps given below'

Once all the details are filled click on the `Update` button to save the details. Now you are all set to send email using SMTP.

### For `Email Password` follow the steps below.

- Visit your `Google Account`
- Click on `Security`
<img src="https://user-images.githubusercontent.com/51023000/150626902-672f1179-a0fd-404c-a77c-a8822ceeb2b8.png" width="200" height="200">

- On this page look for **Sign In to Google** section.

<img src="https://user-images.githubusercontent.com/51023000/150627022-9d4a4c30-ac29-41b4-81a9-3e64e00891f7.png" height="200">

- If you do not see **App Passwords**. Then enable the **2-Step Verification**. Once it is enable click on the **App Passwords**.

- You'll be then redirected to the `App Password` page. Select the **Other(Custom Name)** option while selecting app.

<img src="https://user-images.githubusercontent.com/51023000/150627157-71fd3303-7623-45ce-ac45-0e810b17c0f5.png" height="200">

- Enter the app name and then click on **Generate**

<img src="https://user-images.githubusercontent.com/51023000/150627174-cf2209a0-6cb4-428c-a9ec-7e5a3b25eea5.png" height="200">

- The password will be generated and shown in the dialog box as below.

<img src="https://user-images.githubusercontent.com/51023000/150627202-afa375ae-42b7-4529-b7f5-449f2981eb28.png" height="200">

- Copy the password and paste it in the **Email Password** field in Jamku's SMTP config. And then click on **Update** button to save the config.

**Comment by zero-kishan** [comment]
If the error is 
Error: self-signed certificate

enter the port as 465

**Comment by yogeshkudikala** [comment]
Smtp for outlook

SMTP server name - smtp-mail.outlook.com
SMTP port - 587
and outlook email and password
if mfa is enable then create an app password and use the same for smtp
